NethermindNode.Tests/Tests/General/StabilityTests.cs

```diff
@@ -0,0 +1,30 @@
+
+using NethermindNode.Core.Helpers;
+
+namespace NethermindNode.Tests.SyncingNode;
+
+[TestFixture]
+[Parallelizable(ParallelScope.All)]
+public class StabilityTests : BaseTest
+{
+    private static readonly NLog.Logger Logger = NLog.LogManager.GetLogger(TestContext.CurrentContext.Test.Name);
+
+    [Test]
+    [Category("StabilityCheck")]
+    public void ShouldVerifyIfNodePerformsStable()
```

Comment:
minor, with idea to explain the test logic better
```suggestion
    public void ShouldVerifyThatNodeSyncsWithoutErrors()
```